tests/test_utils.py

```diff
@@ -2139,27 +2139,27 @@ def test_deploy_blacklist_to_constraints():
     assert actual == expected_constraints
 
 
-def test_validate_service_instance_valid_marathon():
-    mock_marathon_instances = [("service1", "main"), ("service1", "main2")]
+def test_validate_service_instance_valid_kubernetes():
+    mock_kubernetes_instances = [("service1", "main"), ("service1", "main2")]
     my_service = "service1"
     my_instance = "main"
     fake_cluster = "fake_cluster"
     fake_soa_dir = "fake_soa_dir"
     with mock.patch(
         "paasta_tools.utils.get_service_instance_list",
         autospec=True,
-        side_effect=[mock_marathon_instances],
+        side_effect=[mock_kubernetes_instances],
     ):
         assert (
             utils.validate_service_instance(
                 my_service, my_instance, fake_cluster, fake_soa_dir
             )
-            == "marathon"
```

Comment:
i'm sorta confused here: how come this isn't `kubernetes`? seems like an interesting choice for us to have had a test that just tested what the first item in a tuple was